tests/test_emulators.py

```diff
@@ -133,11 +398,14 @@ def test_run(self):
         )
         d = model.diagnostics
 
-        np.testing.assert_almost_equal(d["DIAG_NASH_SUTCLIFFE"], -0.117301, 2)
+        np.testing.assert_almost_equal(d["DIAG_NASH_SUTCLIFFE"], -0.116971, 2)
```

Comment:
TODO: check this one!

Reply:
This difference was due to lat/lon values of 0 in the `:Gauge` of the RVT.

Reply:
ok. Great. Maybe we should then test for 4 digits and not just 2 to catch this...

Reply:
Using now same reference `"DIAG_NASH_SUTCLIFFE"` everywhere in code (i.e. `-0.116971`) even though it is the wrong one. The `Latitude`, `Longitude`, and `Elevation` of the entire basin needs to be passed on to `Gauge` as it was done before using:

```
kwargs = dict(
            area=4250.6,
            elevation=843.0,
            latitude=54.4848,
            longitude=-123.3659,
            params=(0.529, -3.396, 407.29, 1.072, 16.9, 0.947),
        )
```

These values are for the entire BASIN and not for a sub-basin or HRU... After that all `"DIAG_NASH_SUTCLIFFE"` should be `-0.117301`.